native/src/App.tsx

```diff
@@ -80,11 +79,14 @@ export const NavigationContainerWithTheme = ({ onStateChange }: NavigationContai
   }
 
   return (
-    <NavigationContainer onStateChange={onStateChange} theme={navigationTheme} linking={linking}>
-      <HeaderButtonsProvider stackType='native'>
-        <Navigator />
-      </HeaderButtonsProvider>
-    </NavigationContainer>
+    <SafeAreaView style={{ flex: 1, backgroundColor: theme.colors.backgroundAccentColor }} edges={['bottom']}>
```

Comment:
🤔 I am actually not sure if we should not rather use the standard background color such that it blends together. Not really sure though.

Reply:
Could you check if you find anything official, e.g. in the material design guidelines?

Reply:
I couldn't find anything about the SafeArea and how it looks with the contrast theme.